src/TransWithoutContext.js

```diff
@@ -3,6 +3,11 @@ import HTML from 'html-parse-stringify';
 import { isObject, isString, warn, warnOnce } from './utils.js';
 import { getDefaults } from './defaults.js';
 import { getI18n } from './i18nInstance.js';
+import { Fragment, isValidElement, cloneElement, createElement, Children } from 'react';
```

Comment:
Why are you importing these again? it's already imported above

Reply:
my bad, not sure what happened here. reset the commits and tried again, it seems that some process adds these during the commit, could it be some git hooks?

Reply:
there is no such magic...
but also remove this and try again: "trim_trailing_whitespace = true"

Reply:
tried with -n option, still the same, guessing some issue with my ide. About trim whitespaces, if it's enabled, whenever you edit tests the trailing spaces gets removed in html elements, and a lot of tests fail because of that, should i still enable it?

Reply:
this does not happen on my machine

Reply:
probably intellij issue then, ill restore it

Reply:
fixed imports, and restore trailing spaces